locales/en/messages.json

```
@@ -3721,10 +3724,16 @@
         "message": "Dynamic Notch Q"
     },
     "pidTuningDynamicNotchMinHz": {
-        "message": "Dynamic Notch Min Hz"
+        "message": "Dynamic Notch Min [Hz]"
```

All the others elements contain the `Frequency` word, so better
```suggestion
        "message": "Dynamic Notch Min Frequency [Hz]"
```

---

Done!